src/pages/clouddocs/release-notes.md

```diff
@@ -22,6 +22,13 @@ subcollection: wh-acd
 
 The following sections document the new features and changes that were included for each release of the Annotator for Clinical Data service.
 
+## January 2023
+
+- Added support for UMLS 2022AA. UMLS 2019AA has been removed and 2020AA is now deprecated (will be removed in 2023 when we add support for UMLS 2023AA).
+- Added support to control whether the Hypothetical service uses internal triggers.
+- Added support for the ACD debug flag in the ACD Python SDK.
+- Updated the documentation for installing and upgrading the Container Edition.
```

Comment:
I don't think the last bullet is necessary. We are continually updating documentation.